suite-common/wallet-utils/src/__fixtures__/antiFraud.ts

```diff
@@ -127,6 +127,30 @@ export const getIsFakeTokenPhishingFixtures = [
         } as TokenDefinitions,
         result: false,
     },
+    {
+        testName: 'one fake, one zero-value legit token tx',
+        transaction: {
+            symbol: 'matic',
+            amount: '0',
+            tokens: [
+                { standard: 'ERC20', contract: '0xA', anmount: '1' },
```

Comment:
```suggestion
                { standard: 'ERC20', contract: '0xA', amount: '1' },
```

Reply:
please type fixtures so that it does not occur again

not one by one but single record for whole fixture array, I accept `any` or `Partial` for some objects, but would be nice to avoid things like `anmount` 😀